core/dbt/include/jsonschemas/project/0.0.110.json

```diff
@@ -4242,10 +4678,22 @@
         "granularity": {
           "default": "day",
           "type": "string"
+        },
+        "time_ingestion_partitioning": {
```

Comment:
this looks correct! move from the top-level properties to `TimeConfig`, but same level as `granularity` which is expected (example: https://docs.getdbt.com/reference/resource-configs/bigquery-configs)